packages/webpack-cli/src/webpack-cli.ts

```diff
@@ -517,6 +560,49 @@ class WebpackCLI implements IWebpackCLI {
     return info;
   }
 
+  async getConfigOutput(options: WebpackDevServerOptions): Promise<void> {
+    const config = await this.loadConfig({
+      config: [options.file ? options.file : "webpack.config.js"],
+      name: options.name,
+      configName: options.name ? (options.name as unknown as string[]) : undefined,
+    });
+    if (options.name) {
+      config.options = Array.isArray(config.options)
+        ? config.options.filter((option: any) => options.name?.includes(option.name))
+        : [];
+    }
+
+    if (options.include) {
+      const includeOptions = webpack.config.getNormalizedWebpackOptions({});
```

Comment:
Also I think we need to apply defaults, so you will see how webpack sees your configuration

Reply:
Actually we need to comment the default values beside them which is getting very complex so shall I apply the defaults?

Reply:
You can get default value `applyWebpackOptionsBaseDefaults` and `applyWebpackOptionsDefaults` from:
```
const {
	applyWebpackOptionsDefaults,
	applyWebpackOptionsBaseDefaults
} = require("webpack/config/defaults");
```

How I see this logic, you load config file/files, normalize and them apply default.

Also we have two possible commands:
- `webpack config` just print full resolved configuration, i.e. after apply normalization and default
- `webpack config --diff` show only options which are different after normalize and default, so you can undestand which options you don't need because they are default